Source/webserver/src/main/java/klingon/webserver/WebserverApplication.java

```diff
@@ -3,22 +3,135 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.event.ContextStartedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.scheduling.annotation.*;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
+import org.springframework.stereotype.Service;
 
 import java.util.List;
 
+/**
+ * This class is a initializer to boot up the applications backend
+ * and handles different execution of the built in database
+ *
+ *  * @author Anthon Lenander, Phong Nguyen
+ *  * @version 2021-09-21
+ *
+ */
+
+@Service
 @SpringBootApplication
 public class WebserverApplication
 {
+	private static BicycleStationRepository bicycleStationRepository;
+	private static PumpStationRepository pumpStationRepository;
+
 	public static void main(String[] args)
 	{
 		ConfigurableApplicationContext context = SpringApplication.run(WebserverApplication.class, args);
-    	BicycleStationRepository bicycleStationRepository = context.getBean(BicycleStationRepository.class);
-		PumpStationRepository pumpStationRepository = context.getBean(PumpStationRepository.class);
+    	bicycleStationRepository = context.getBean(BicycleStationRepository.class);
```

Comment:
Fix indentation

Reply:
I fixed the indentation, but it seemed to be more of an issue with how github displays your code than with the actual code being miss-indented though...